src/environment.js

```diff
@@ -324,7 +325,9 @@ var Environment = Obj.extend({
 
         var tmpl = new Template(src, this, opts.path);
         return tmpl.render(ctx, cb);
-    }
+    },
+
+    waterfall: waterfall
```

Comment:
I doubt that it's good enough way to include lib code into compiled template. Please feel free to suggest a better one